### PR TITLE
Gallery  component

### PR DESCRIPTION
**Pull request by 3lvis**
Added a `GalleryComponent`, it's composed of an array of `stringURLs` elements.

You can initialize a `GalleryComponent` like this:

```swift
let gallery = GalleryComponent(placeholder: imagePlaceholder, stringURLs: ["https://images.finncdn.no/dynamic/480x360c/2017/9/vertical-5/30/5/105/424/_1263219766.jpg", "https://images.finncdn.no/dynamic/480x360c/2017/7/vertical-2/19/3/100/464/_1229205040.jpg"])
```

gallery

### CODE REVIEW

File: Sources/Components/Gallery/GalleryComponent.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+
+import Foundation
+
+public struct GalleryComponent: Component {
+    public let id: String
+    public let placeholder: UIImage
+    public let stringURLs: [String]
+
+    public init(id: String = UUID().uuidString, placeholder: UIImage, stringURLs: [String]) {
```
Comment (ulrikus) [inline Sources/Components/Gallery/GalleryComponent.swift:12]: Could the placeholder image possibly have a default vale as the asset you added? Or do we not want it to hold images..?

File: Sources/Components/Gallery/GalleryComponentView.swift [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var horizontalStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.distribution = .fillProportionally
+
```
Comment (ulrikus) [inline Sources/Components/Gallery/GalleryComponentView.swift]: Nit-picking, but maybe remove unnecessary white space? :)

File: Sources/Components/Gallery/GalleryComponent.swift (hunk 1 above)
Comment (3lvis) [reply Sources/Components/Gallery/GalleryComponent.swift:12]: I think it's better to not have a default placeholder.

File: Sources/Components/Gallery/GalleryComponentView.swift (hunk 2 above)
Comment (3lvis) [reply Sources/Components/Gallery/GalleryComponentView.swift]: Fixed :)

**Review by ulrikus — APPROVED** [review]
I liked this one. Neet! ;)

**Pull request merged**